Amplify/Categories/DataStore/Model/Temporal/TemporalSpec.swift

```diff
@@ -0,0 +1,9 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
Empty file?

Reply:
This was moved into `Temporal.swift` to help with the diff. Deleting it.

Reply:
Addressed in [cdace94](https://github.com/aws-amplify/amplify-ios/pull/1760/commits/cdace94592eca4ee9c74d5ac7b52811187b179c1)